.github/workflows/build-runs-on-image.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build runs-on image
+
+on:
```

Comment:
as a quick hack to test this on your PR, you can probably add the `push` trigger on specifically your branch `aluon/runs-on-image-workflow` :) make sure to remove it after you've confirmed it works though, so you don't accidentally land it

This way you don't have to wait until landing in main to try `workflow_dispatch`